Code/Framework/AzCore/AzCore/Jobs/Internal/JobManagerWorkStealing.cpp

```diff
@@ -370,7 +369,8 @@ void JobManagerWorkStealing::ProcessJobsInternal(ThreadInfo* info, Job* suspende
                 if (shouldSleep)
                 {
                     //no available work, so go to sleep (or we have already been signaled by another thread and will acquire the semaphore but not actually sleep)
-                    info->m_waitEvent.acquire();
+                    AZStd::unique_lock<AZStd::mutex> lock{ info->m_waitMutex };
```

Comment:
FYI Class Template Argument Deduction gets rid of the need to specify `<AZStd::mutex>`

Reply:
Thanks will change I keep forgetting we're on C++17 (worked on a C++14 codebase most recently). CTAD is indeed great